pkg/server/list_users_test.go

```diff
@@ -231,12 +249,39 @@ func TestExperimentalListUsers(t *testing.T) {
 		require.Equal(t, codes.Unimplemented, e.Code())
 	})
 
-	t.Run("list_users_does_not_error_if_experimentally_enabled", func(t *testing.T) {
+	t.Run("list_users_returns_error_if_latest_model_not_found", func(t *testing.T) {
+		mockDatastore.EXPECT().FindLatestAuthorizationModel(gomock.Any(), gomock.Any()).Return(nil, storage.ErrNotFound) // error demonstrates that main code path is reached
+
 		server.experimentals = []ExperimentalFeatureFlag{ExperimentalEnableListUsers}
```

Comment:
nit: the parent test is `TestExperimentalListUsers` so i feel this line could be moved up